Dynatrace/dynatrace-operator
# Add image import to snyk to publish image workflow

# Description

Automatically import images of new releases into our Synk integration.

## How can this be tested?
I've tested this as good as thoroughly as possible on my private repo fork. It worked fine there.

You need to create a new release tag, see if the workflow run of "publish_image" was succesful and check in Snyk if the new image is listed.

## Checklist
- [n/a] Unit tests have been updated/added
